football/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration packages="com.amazonaws.services.lambda.runtime.log4j2">
+    <Appenders>
+        <Lambda name="Lambda">
+            <PatternLayout>
+                <pattern>%d{yyyy-MM-dd HH:mm:ss} [%t] %X{AWSRequestId} %-5p %c{1}:%L - %m%n</pattern>
```

Comment:
What do the bits that come after `AWSRequestId` do?

Reply:
`2020-09-03 08:02:19 [main] 5dca5d19-98d4-4fb4-964d-15296ed02047 INFO Lambda$:108 - Finished processing`

`%-5p` = Priority of logging event (justified to a width of five characters)
`%c{1}:%L` = Abbreviated logger name (i.e. class name) and line number
`%m%n` = Message + line separator

(I just [learnt](https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternLayout) all of this too, I copied and pasted this file from another lambda in this repo 🙈)